### PR TITLE
Remove snaps in goto & nearby teleport references

### PR DESCRIPTION
**Pull request by digisomni**
This addresses #1021 in that the Pal's text was not updated to reflect the change.

Another PR should be opened later to perhaps come up with a better way to indicate someone is not available or have it go from Interface -> Domain -> Other Person's Interface instead of (I believe) traversing the Metaverse.

The need to have people be able to effortlessly teleport and follow people around as newbies is a priority. We should come up with a good privacy indicator/function for within domains that doesn't affect this in any substantial way.

P.S. didn't mean to open this branch in the main repo, GitHub web editor, amirite?

### CODE REVIEW

**Comment by vircadia-build-notifier** [comment]
The following links are available: 
**build (macOS-latest, client)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1298/Vircadia-Interface-PR1298-becaa21-_facee334.dmg

**build (macOS-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1298/Vircadia-PR1298-becaa21-_facee334.dmg

**build (ubuntu-18.04, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1298/Vircadia-PR1298-becaa21-_facee334.sh
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1298/Vircadia-PR1298-becaa21-.tar_facee334.Z
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1298/Vircadia-PR1298-becaa21-.tar_facee334.gz

**build (windows-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1298/Vircadia-PR1298-becaa21-_facee334.exe

**Review by ctrlaltdavid — APPROVED** [review]

**Review by daleglass — APPROVED** [review]

**Review by daleglass — CHANGES_REQUESTED** [review]
This change leaves some strings ending with a `,`

File: interface/resources/qml/hifi/Pal.qml [hunk 1]
```diff
@@ -1109,10 +1109,10 @@ Rectangle {
...
-                        "Your location will be visible in the 'Connections' list only for those with whom you are friends. They'll be able to jump to your location if the domain allows, and you will see 'Snaps' Blasts from them in 'Go To'",
-                        "You will appear offline in the 'Connections' list, and you will not receive Snaps Blasts from connections or friends in 'Go To'."],
+                        ["Your username will be visible in everyone's 'Nearby' list.",
+                        "Your location will be visible in the 'Connections' list only for those with whom you are connected or friends. They'll be able to jump to your location if the domain allows,",
+                        "Your location will be visible in the 'Connections' list only for those with whom you are friends. They'll be able to jump to your location if the domain allows,",
+                        "You will appear offline in the 'Connections' list,"],
```
Comment (ctrlaltdavid) [inline interface/resources/qml/hifi/Pal.qml:1115]:
```suggestion
                        ["Your username will be visible in everyone's 'Nearby' list.",
                        "Your location will be visible in the 'Connections' list only for those with whom you are connected or friends. They'll be able to jump to your location if the domain allows.",
                        "Your location will be visible in the 'Connections' list only for those with whom you are friends. They'll be able to jump to your location if the domain allows.",
                        "You will appear offline in the 'Connections' list."],
```

**Review by ctrlaltdavid — CHANGES_REQUESTED** [review]

**Comment by vircadia-build-notifier** [comment]
The following links are available: 
**build (macOS-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1298/Vircadia-PR1298-336ba38-_3f1c7fae.dmg

**build (macOS-latest, client)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1298/Vircadia-Interface-PR1298-336ba38-_3f1c7fae.dmg

**build (ubuntu-18.04, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1298/Vircadia-PR1298-336ba38-.tar_3f1c7fae.Z
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1298/Vircadia-PR1298-336ba38-.tar_3f1c7fae.gz
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1298/Vircadia-PR1298-336ba38-_3f1c7fae.sh

**build (windows-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1298/Vircadia-PR1298-336ba38-_3f1c7fae.exe

**Review by ctrlaltdavid — APPROVED** [review]

**Comment by vircadia-build-notifier** [comment]
The following links are available: 
**build (ubuntu-18.04, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1298/Vircadia-PR1298-c33b2ad-.tar_3f1c7fae.gz
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1298/Vircadia-PR1298-c33b2ad-_3f1c7fae.sh
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1298/Vircadia-PR1298-c33b2ad-.tar_3f1c7fae.Z

**build (windows-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1298/Vircadia-PR1298-c33b2ad-_3f1c7fae.exe

**build (macOS-latest, client)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1298/Vircadia-Interface-PR1298-c33b2ad-_3f1c7fae.dmg

**build (macOS-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1298/Vircadia-PR1298-c33b2ad-_3f1c7fae.dmg

**Comment by digisomni** [comment]
It works correctly in testing. However, now the menu has whitespace at the bottom due to the smaller button sizes.

**Comment by digisomni** [comment]
This PR will be updated by Namark to fix the spacing then it should be good to review and test once more before merging.

**Comment by namark** [comment]
Couldn't quite figure out how to make it sized based on content, so just eyeballed it.

**Comment by vircadia-build-notifier** [comment]
The following links are available: 
**build (macOS-10.15, client)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1298/Vircadia-Interface-PR1298-89d4163-_f9eb292e.dmg

**build (macOS-10.15, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1298/Vircadia-PR1298-89d4163-_f9eb292e.dmg

**build (ubuntu-18.04, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1298/Vircadia-PR1298-89d4163-.tar_f9eb292e.gz
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1298/Vircadia-PR1298-89d4163-.tar_f9eb292e.Z
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1298/Vircadia-PR1298-89d4163-_f9eb292e.sh

**Review by namark — APPROVED** [review]

**Comment by vircadia-build-notifier** [comment]
The following links are available: 
**build (macOS-10.15, client)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1298/Vircadia-Interface-PR1298-89d4163-_f9eb292e.dmg

**build (macOS-10.15, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1298/Vircadia-PR1298-89d4163-_f9eb292e.dmg

**build (ubuntu-18.04, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1298/Vircadia-PR1298-89d4163-.tar_f9eb292e.gz
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1298/Vircadia-PR1298-89d4163-.tar_f9eb292e.Z
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1298/Vircadia-PR1298-89d4163-_f9eb292e.sh

**build (windows-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1298/Vircadia-PR1298-89d4163-_f9eb292e.exe

**Pull request merged**